src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2996,39 +2996,53 @@ public void ParseOut2()
         public void ParseInstrumentTestNames()
```

Comment:
>ParseInstrumentTestNames [](start = 20, length = 24)

It looks like we are not testing scenarios when /instrument switch is specified multiple times.